xline/src/storage/kvwatcher.rs

```diff
@@ -246,7 +263,9 @@ impl KvWatcherInner {
         let initial_events = if start_rev == 0 {
             vec![]
         } else {
-            self.storage.get_event_from_revision(key_range, start_rev)
+            self.storage
```

Comment:
Shall we at least give a warning here? I guess it's not normal if we fail to get events from storage?

Reply:
fixed